src/test/java/com/at/stepdefinitions/Get.java

```diff
@@ -1,51 +1,142 @@
 package com.at.stepdefinitions;
 
-import com.at.globalclasses.ApiPaths;
-import com.at.globalclasses.ApiTools;
-import com.at.globalclasses.BasicSecurityUtil;
+import com.at.globalclasses.*;
 import io.cucumber.java.en.Given;
 import io.cucumber.java.en.Then;
 import io.cucumber.java.en.When;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import gherkin.deps.com.google.gson.JsonParser;
 import org.junit.Assert;
 
 public class Get {
     private BasicSecurityUtil base;
+    private BasicSecurityUtil basedb;
```

Comment:
What is the difference between this property and the one above?

Reply:
"base" take the info to make the GET request.
and "basedb" take the info to implement connection with the DataBase